### ISSUE TITLE
Same problem with multiple Rockstar games on Play!

### ISSUE THREAD

**Issue by shammalama** [jpd002/Play-#256]
All Rockstar games I've tried on Play! seem to have the same exact issue: they load, show the intro movies, but if you try and skip them, there's freezing; and if you let them play through, they finish but then hang on a black screen and never move on. Here are all the games that do the same thing:

1.Grand Theft Auto: Vice City.

2.Grand Theft Auto: San Andreas

3.Grand Theft Auto: Vice City Stories

4.Bully.

I'm guessing that, since this issue seems the same between all of them (maybe it's the same exact game engine that's doing the same thing that Play! doesn't like?), you can kill 4 birds with 1 stone.

In other words, fixing the issues with just one of these games will likely get the rest to work as well.

**Comment by jpd002** [comment]
Indeed, it's probably the same issue with all those games. I'll need to get one of those and take a look at what's going on.
